SystemAudioPlayer/CHANGELOG.md

```diff
@@ -11,4 +11,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Change
 - Reset API version to 1.0.0
-- Change README to inform how to update changelog and API version
\ No newline at end of file
+- Change README to inform how to update changelog and API version
+
+## [1.0.1]
+### Fixed
```

Comment:
Pls move this to the top. Latest version should be on top. Also remove JIRA ticket numbers from changelog. Would be good to add this note to all the changelogs.